lib/health-data-standards/models/record.rb

```diff
@@ -60,15 +60,11 @@ class Record
   scope :by_provider, ->(prov, effective_date) { (effective_date) ? where(provider_queries(prov.id, effective_date)) : where('provider_performances.provider_id'=>prov.id)  }
   scope :by_patient_id, ->(id) { where(:medical_record_number => id) }
 
-  def self.update_or_create(data)
```

Comment:
Can you leave the existing method and add `delete_and_create` separately? Will make future merging of updates easier.